storage/integration_test.go

```diff
@@ -7111,6 +7112,7 @@ func (h testHelper) mustDeleteBucket(b *BucketHandle) {
 
```

Comment:
Looks like you applied the option to some of these helper methods but not others? why not do it across the board?

Reply:
I didn't want to take the time to do a thorough review right now (I am not sure if all of these functions even retry). We should move these functions out into a testutil and consolidate some of them.